### PR TITLE
feat(action): build debian package for arm64 arch

### PR DESCRIPTION
**Pull request by boazsade**
Signed-off-by: Boaz Sade <boaz@dragonflydb.io>

This would add support for building the debian package for ARM64 based arch.
* The git action file will run the script that generate a package for ARM as well as for x86.
* The build script is now accepting new arg of the arch type.
* The build of the ARM package can only be run inside environment that is itself ARM64.

### CODE REVIEW

File: tools/packaging/generate_debian_package.sh [hunk 1]
```diff
@@ -12,11 +12,23 @@
...
+# Params:
+#	* arch type: either amd64 or arm64 (aarch64)
+#	* optional location to the binary to place at the package
 
 
 set -eu
 
+ARCH_VAL=amd64
+if [ $# -eq 1 ]; then
+	if [ "$1" != "${ARCH_VAL}" -a "$1" != "aarch64" ]; then
```
Comment (romange) [inline tools/packaging/generate_debian_package.sh:25]: you do not need to pass it by argument. you can do it with `uname -m`
See here for example: 
https://github.com/romange/image-bakery/blob/d8785cb97b20950ca799f06df63a9042f5bf4c79/provision/cloudconfig.cue#L81

**Review by romange — COMMENTED** [review]
lgtm but please make the architecture deduction automatic based on uname

File: tools/packaging/generate_debian_package.sh (hunk 1 above)
Comment (boazsade) [reply tools/packaging/generate_debian_package.sh:25]: OK, I will just make sure that this is working (actually we just need to make sure that this not x86 and set it to aarch64)

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -77,6 +77,8 @@ jobs:
               done
             fi
             ./tools/release.sh
+            ./tools/packaging/generate_debian_package.sh aarch64 build-opt/dragonfly-aarch64
```
Comment (romange) [inline .github/workflows/release.yml:80]: @boazsade  no need to pass architecture argument anymore

Comment (boazsade) [reply .github/workflows/release.yml]: you're right!

**Review by romange — APPROVED** [review]

**Pull request merged**